openpype/hosts/photoshop/plugins/create/create_image.py

```diff
@@ -163,6 +165,12 @@ def _handle_legacy(self, instance_data):
     def _clean_highlights(self, stub, item):
         return item.replace(stub.PUBLISH_ICON, '').replace(stub.LOADED_ICON,
                                                            '')
-    @classmethod
-    def get_dynamic_data(cls, *args, **kwargs):
-        return {"layer": "{layer}"}
+
+    def get_dynamic_data(self, variant, task_name, asset_doc,
+                         project_name, host_name, instance):
+        if not instance:
+            return {"layer": "{layer}"}
+        else:
+            layer_name = instance.get("layer_name")
+            if layer_name:
+                return {"layer": layer_name}
```

Comment:
There is no default output
```suggestion
        if instance is not None:
            layer_name = instance.get("layer_name")
            if layer_name:
                return {"layer": layer_name}
        return {"layer": "{layer}"}
```